app/src/main/java/com/oracle/javafx/scenebuilder/app/welcomedialog/WelcomeDialogWindowController.java

```diff
@@ -170,6 +170,14 @@ private void loadAndPopulateRecentItemsInBackground() {
         t.start();
     }
 
+    private void setupTemplateTooltips() {
+        Tooltip.install(emptyApp, new Tooltip(I18N.getString("template.description.new.empty.app")));

```

Comment:
For maintainability, it might be better to use a loop through `apps` and related `templates`

Reply:
Actually, I've been thinking to add the tooltip text to #516 

It seems each _app_ button has `userData = template`, so then the loop becomes trivial. Happy to refactor here, but #516 should be merged first to avoid merge issues.

Reply:
Also, in my opinion, there is enough space for template descriptions to be presented permanently, just below the title. The title can use a bold(er) font and wrapped text below with regular one, for example